ethereum/contracts/bridge/Bridge.sol

```diff
@@ -79,6 +79,9 @@ contract Bridge is BridgeGovernance, ReentrancyGuard {
         uint256 arbiterFee,
         uint32 nonce
     ) public payable returns (uint64 sequence) {
+        require(recipient != bytes32(0), "invalid recipient");
```

Comment:
It is not prohibited for an EVM chain or another platform’s token bridge implementation to use the zero address as a valid recipient